integration/project.yaml

```diff
@@ -4,14 +4,17 @@ test:
   env:
     - "VAULT_TOKEN=mytoken"
     - "CIRCLE_SHA1=${CIRCLE_SHA1}"
-    - "COMMON_DOMAIN=${COMMON_DOMAIN}"
+    - "COMMON_DOMAIN_GUEST=${COMMON_DOMAIN_GUEST}"
     - "CLUSTER_NAME=${CLUSTER_NAME}"
     - "TESTED_VERSION=${TESTED_VERSION}"
     - "REGISTRY_PULL_SECRET=${REGISTRY_PULL_SECRET}"
     - "IDRSA_PUB=${IDRSA_PUB}"
-    - "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}"
-    - "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}"
-    - "AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}"
+    - "AWS_ACCESS_KEY_ID_GUEST=${AWS_ACCESS_KEY_ID_GUEST}"
+    - "AWS_SECRET_ACCESS_KEY_GUEST=${AWS_SECRET_ACCESS_KEY_GUEST}"
+    - "AWS_SESSION_TOKEN_GUEST=${AWS_SESSION_TOKEN_GUEST}"
+    - "AWS_ACCESS_KEY_ID_HOST=${AWS_ACCESS_KEY_ID_HOST}"
+    - "AWS_SECRET_ACCESS_KEY_HOST=${AWS_SECRET_ACCESS_KEY_HOST}"
+    - "AWS_SESSION_TOKEN_HOST=${AWS_SESSION_TOKEN_HOST}"
```

Comment:
It wold be easier for my eyes if those were prefixes like `HOST_AWS_`, but it isn't a blocker.